main/boost/APKBUILD

```diff
@@ -153,28 +161,42 @@ package() {
 }
 
 static() {
-	pkgdesc="Static libraries for boost"
+	pkgdesc="Boost static libraries"
+	depends="$depends_static"
```

Comment:
where is `depends_static` defined? if it not used other place, then we can remove this line.

Reply:
It's not currently defined. However, we can't remove this line without inheriting the depends for `boost` when `boost-static` is created.